### PR TITLE
Copy only last 5MB of data from /var/log/messages file

### PR DESCRIPTION
**Pull request by awsvikrant**
Submitting PR for #12. This PR has changes to collect last 5MB of /var/log/message instead of everything.

### CODE REVIEW

**Comment by nithu0115** [comment]
Thanks @awsvikrant.

I tested it and is working as expected. 

Lets wait for @dkeightley to look into it before we merge.

**Comment by dkeightley** [comment]
Working for me as expected also, `tail -c` is POSIX compliant which is great.

LGTM!

**Review by nithu0115 — APPROVED** [review]
Ship it

**Pull request merged**
